modules/sdk-coin-trx/test/unit/index.ts

```diff
@@ -1,14 +1,15 @@
-import should from 'should';
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
 import { getBuilder } from '../../src/lib/builder';
 
 describe('Coin factory', () => {
   it('should instantiate TRX builder properly', () => {
     const trxBuilder = getBuilder('trx');
-    should.exist(trxBuilder);
+    assert.notEqual(trxBuilder, undefined);
```

Comment:
think you should use assert.ok for this https://nodejs.org/api/assert.html#assertokvalue-message

Reply:
for exist checks